### DISCUSSION TITLE
Custom domain

### DISCUSSION THREAD

**Discussion by zhex900** [directus/directus#14168] [Feature Requests]
Hi,

It will be good to add a custom domain for a project.

I have added a redirect from cloudfront. But I cannot login. I guess there is origin check. 

```{
  "errors": [
    {
      "message": "Invalid user credentials.",
      "extensions": {
        "code": "INVALID_CREDENTIALS"
      }
    }
  ]
}
```

Jake

**Comment by rijkvanzanten** [comment]
Are you talking about Directus in general, or Directus Cloud?

**Comment by zhex900** [comment]
Directus Cloud. Like Netlify, you can add your own custom domain. 

I know how to have a custom domain for self-hosting.

**Reply by benhaynes** [reply]
You can customize the slug on standard cloud, and I believe we're close to fully custom domains on Enterprise cloud. CC @wrynegade

**Reply by etmartinkazoo** [reply]
We have a few client projects that would be a great fit for the "Standard" cloud option but are self hosting for now because a custom domain is important to the client. When this is made available we will happily move the infrastructure/server side of things your way. Great product, thank you!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
